internal/ingress/controller/config/config.go

```diff
@@ -393,9 +393,17 @@ type Configuration struct {
 	// upstream servers that are preserved in the cache of each worker process. When this
 	// number is exceeded, the least recently used connections are closed.
 	// http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive
-	// Default: 32
 	UpstreamKeepaliveConnections int `json:"upstream-keepalive-connections,omitempty"`
 
+	// Sets a timeout during which an idle keepalive connection to an upstream server will stay open.
+	// http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive_timeout
+	UpstreamKeepAliveTimeOut int `json:"upstream-keepalive-timeout,omitempty"`
```

Comment:
please use `Timeout` instead of `TimeOut`, the latter is not consistent with the naming style in the rest of code base

Reply:
same goes to `KeepAlive` -> `Keepalive`

Reply:
Makes sense for consistency.